libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -303,9 +309,11 @@ export class WorkspaceInstallManager extends BaseInstallManager {
       }
     }
 
-    const doInstall = (options: IInstallManagerOptions): void => {
+    const doInstall = async (options: IInstallManagerOptions): Promise<void> => {
```

Comment:
```suggestion
    const doInstallAsync = async (options: IInstallManagerOptions): Promise<void> => {
```

Reply:
I changed to `doActualInstallAsync`, what do you think? I didn't name it `doInstallAsync` because there is already a public `doInstallAsync` function in the `WorkspaceManager` class.

Reply:
Renamed to `doInstallInternalAsync`.